src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -25,17 +25,23 @@
                 <div class="col-md-2 project-cell">
                     <a @click="deleteProject($event, p.id)" href="" v-translate="'delete'"></a>
                 </div>
+                <div class="col-md-2 project-cell">
+                    <a @click="copyProject($event, p.id)" href="" v-translate="'copyToNewProject'"></a>
```

Comment:
Coping whole projects will be a bit more involved than just copying a single version into a new project, and will almost certainly require some back-end work. Is it part of the spec, or is copying a single version into a new project enough to satisfy requirements at this point? I'd be tempted to leave whole project copying for now, and wait to see if it's needed.